test/src/main/java/org/apache/rocketmq/test/util/MQAdminTestUtils.java

```diff
@@ -17,10 +17,15 @@
 
 package org.apache.rocketmq.test.util;
 
+import java.util.HashMap;
```

Comment:
ditto

Reply:
I use idea tool to fromat the code, so maybe change the the order of the previous packages. I think when need to remove the useless import packages,The functionality of idea is recommended instead of manual operation

Reply:
It will be OK if the sort is automatic